plutus-tx/src/PlutusTx/Builtins.hs

```diff
@@ -195,13 +220,27 @@ verifyEcdsaSecp256k1Signature vk msg sig =
 -- and a message (all as 'BuiltinByteString's), verify the message with that key
 -- and signature.
 --
--- = Important note
+-- = Note
+--
+-- There are additional well-formation requirements for the arguments beyond
+-- their length:
+--
+-- * The bytes of the public key must correspond to the /x/ coordinate, as a
+-- big-endian integer, as specified in BIP-340.
+-- * The first 32 bytes of the signature must correspond to the /x/ coordinate,
+-- as a big-endian integer, as specified in BIP-340.
+-- * The last 32 bytes of the signature must correspond to the bytes of /s/, as
+-- a big-endian integer, as specified in BIP-340.
+--
+-- = See also
 --
--- The verification key and signature must all be of appropriate form and
--- length. This function will error if this is not the case.
+-- * [BIP-340](https://github.com/bitcoin/bips/blob/master/bip-0340.mediawiki)
+-- *
+-- [@secp256k1_xonly_pubkey_serialize@](https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1_extrakeys.h#L61);
+-- this implements the format for the verification key that we accept.
```

Comment:
Maybe it would also be useful to point out the function that 'creates' the Schnorr signature as we accept it, i.e. `secp256k1_schnorrsig_sign`.

Reply:
Would it also make sense to document this information for the ECDSA scheme?

Reply:
I think the comment saying that we use `secp256k1_ecdsa_serialize_compact` is sufficient in that case.